src/platform/ios.rs

```diff
@@ -1,7 +1,25 @@
 //! # iOS / UIKit
 //!
-//! Winit has an OS requirement of iOS 8 or higher, and is regularly tested on
-//! iOS 9.3.
+//! Winit has [the same iOS version requirements as `rustc`][rustc-ios-version], although it's
+//! frequently only tested on newer iOS versions.
+//!
+//! [rustc-ios-version]: https://doc.rust-lang.org/stable/rustc/platform-support/apple-ios.html#os-version
+//!
+//! ## Running on Mac Catalyst
+//!
+//! Mac Catalyst allows running applications using UIKit on macOS, which can be very useful for
+//! testing. Winit somewhat supports this, though you'll need to bundle your application before
+//! running it.
+//!
+//! To run e.g. the `window` example in the Winit repository, you can use [`cargo-bundle`] as
+//! follows:
+//!
+//! ```console
+//! $ cargo +nightly bundle --format=ios --target=aarch64-apple-ios-macabi --example=window
+//! $ ./target/aarch64-apple-ios-macabi/debug/bundle/ios/window.app/window
```

Comment:
This is not the actual command. The actual command right now is:
```suggestion
//! $ cargo +nightly build -Zbuild-std --target=aarch64-apple-ios-macabi --example window
//! $ CARGO_BUNDLE_SKIP_BUILD=1 cargo bundle --example window --target=aarch64-apple-ios-macabi --format=ios
//! $ ./target/aarch64-apple-ios-macabi/debug/bundle/ios/window.app/window
```

Since `cargo-bundle` doesn't support `-Zbuild-std` and https://github.com/rust-lang/rust/pull/126450 has yet to be merged (once that's in stable we can also remove the `+nightly` from this command).

You'll also need to use a patched version of `cargo-bundle` (or set `[package.metadata.bundle.example.window]`) until [a new version is released](https://github.com/burtonageo/cargo-bundle/pull/185).

Reply:
Aaand both are resolved, so with `cargo-bundle v0.6.1` and the next nightly, that is the command